epub33/core/ECHIDNA

```diff
@@ -0,0 +1,5 @@
+# ECHIDNA configuration
```

Comment:
The Echidna manifest files are not required with spec-prod, so feel free to remove them.

Reply:
Thanks, I will. But is this also true when the document includes external files like figures (some of the specs we have do that)?

Reply:
Yes. It'll find them automatically and copy them, as long as they're in same repo or are images hosted on githubusercontent.com (images in GitHub issues).
You can verify that all images/embedded content is picked in the "build" step of logs.